index.js

```diff
@@ -248,16 +246,9 @@ Dat.prototype.download = function (cb) {
     self.emit('upload', data)
   })
 
-  function updateTotalStats () {
-    self.stats.bytesTotal = archive.content ? archive.content.bytes : 0
-    var fileCount = 0
-    each(archive.list({live: false}), function (data, next) {
-      if (data.type === 'file') fileCount++
-      next()
-    }, function () {
-      self.stats.filesTotal = fileCount
-    })
-  }
+  self._stats.on('update', function () {
+    self.stats = extend(self.stats, self._stats.get())
```

Comment:
I'm doing this because we still need the `filesTotal` and `filesProgress` on the share side to reflect the total count in the directory and how many files have been added to hyperdrive, respectively. Those aren't (and probably shouldn't) be done in hyperdrive-stats.